src/Sulu/Bundle/AdminBundle/Resources/public/js/components/login/main.js

```diff
@@ -1,54 +1,710 @@
 /*
- * This file is part of the Sulu CMS.
+ * This file is part of the Sulu CMF.
  *
- * (c) MASSIVE ART Webservices GmbH
+ * (c) MASSIVE ART WebServices GmbH
  *
  * This source file is subject to the MIT license that is bundled
  * with this source code in the file LICENSE.
+ *
  */
 
 /**
  * @class Login
  * @constructor
  *
- * @params {Object} [options] Configuration object
+ * @param {Object} [options] Configuration object
+ * @param {String} [options.instanceName] The instance name of the sidebar
+ * @param {String} [options.backgroundImg] url to the background image
+ * @param {String} [options.shiftSpace] numbers of pixels on each edge available for moving the image
+ * @param {String} [options.fadeInDuration] fade in duration of the image
+ * @param {String} [options.loginCheck] path to post the login-credentials to
  */
 
-require.config({
-    paths: {
-        '__component__$login@suluadmin': 'components/login/main'
-    },
-    include: [
-        '__component__$login@suluadmin'
-    ]
-});
-
-define(function() {
+define([], function() {
 
     'use strict';
 
-    var createEventName = function(postfix) {
-            return 'sulu.login.' + postfix;
+    var defaults = {
+            instanceName: '',
+            backgroundImg: '/bundles/suluadmin/img/background.jpg',
+            shiftSpace: 50, //px
+            fadeInDuration: 350,
+            loginUrl: '',
+            loginCheck: '',
+            resetMailUrl: '',
+            resendUrl: '',
+            resetUrl: '',
+            resetToken: '',
+            resetMode: false,
+            translations: {
+                resetPassword: 'sulu.login.reset-password',
+                reset: 'public.reset',
+                backLogin: 'sulu.login.back-login',
+                resend: 'sulu.login.resend-email',
+                emailSent: 'sulu.login.email-sent-msg',
+                backWebsite: 'sulu.login.back-website',
+                login: 'public.login',
+                errorMessage: 'sulu.login.error-msg',
+                forgotPassword: 'sulu.login.forgot-password',
+                emailUser: 'sulu.login.email-username',
+                password: 'public.password',
+                emailSentSuccess: 'sulu.login.email-sent-success',
+                enterNewPassword: 'sulu.login.enter-new-password',
+                repeatPassword: 'sulu.login.repeat-password'
+            },
+            errorTranslations: {
+                0: 'sulu.login.mail.user-not-found',
+                1003: 'sulu.login.mail.already-sent',
+                1005: 'sulu.login.token-does-not-exist',
+                1007: 'sulu.login.mail.limit-reached'
+            }
+        },
+
+        constants = {
+            componentClass: 'sulu-login',
+            backgroundClass: 'background',
+            imageClass: 'image',
+            darkenerClass: 'darkener',
+            bgActiveClass: 'active',
```

Comment:
you are tough :)
so. in the front-end are no more abbreviations. i even changed the translation keys ;)